data/title/rct1aa/script.txt

```diff
@@ -1,187 +0,0 @@
-# RollerCoaster Tycoon 1 Title Sequence
-
-##############################
-# Added Attractions          #
-##############################
-# SC_HAUNTED_HARBOR
-LOADRCT1 45
-LOCATION 62, 53
-WAIT 11
-ROTATE 3
-LOCATION 78, 45
-WAIT 16
-
-# SC_CANARY_MINES
-LOADRCT1 42
-LOCATION 52, 28
-WAIT 15
-
-# SC_GENTLE_GLEN
-LOADRCT1 48
-LOCATION 68, 45
-WAIT 11
-
-# SC_FUNTOPIA
-LOADRCT1 44
-LOCATION 69, 93
-WAIT 17
-
-# SC_SPRIGHTLY_PARK
-LOADRCT1 51
-ROTATE 3
-LOCATION 97, 72
-WAIT 12
-ROTATE 2
-LOCATION 74, 71
-WAIT 12
-ROTATE 2
-LOCATION 67, 90
-WAIT 12
-
-# SC_GIGGLE_DOWNS
```

Comment:
That is true. The main reason to switch is because the parser would get much more complicated when doing issues like #4500 for the current format. I guess one (not ideal) solution would be to create special elements that act like comments and are ignored in code?